valhalla/filesystem_utils.h

```diff
@@ -0,0 +1,22 @@
+#include <sys/stat.h>
+
+#include <chrono>
+#include <filesystem>
+
+#ifdef _WIN32
+#define FS_MTIME(st_stat) st_stat.st_mtime
+#elif __APPLE__
+#define FS_MTIME(st_stat) st_stat.st_mtime
+#else
+#define FS_MTIME(st_stat) st_stat.st_mtim.tv_sec
+#endif
+
+namespace std::filesystem {
```

Comment:
probably not a good idea? I did it more out of a lack of other good namespace ideas, but maybe the main `valhalla` namespace is good enough, rather than extending a std class?

Reply:
though I doubt `std::filesystem` will ever implement `last_write_time_t` or `save` (it's designed to _not_ modify or create files AFAIK; well, other than deleting them)